src/server/handlers/github-auth.js

```diff
@@ -114,7 +114,7 @@ export const verify = (req, res, next) => {
     });
 
     // Redirect to logged in URL
-    res.redirect(`/user/${req.session.user.login}`);
+    res.redirect(`/user/${req.session.user.login}?sign-in`);
```

Comment:
Workaround for redirecting users to /select-repositories if they don't have any repos.

We should only do the redirect after they sign in, so to distinguish getting to "My repos" from login or from other part of the app, we can use URL param.

This should not be needed once we implement proper checks if users ever had any repos (#780)